tests/JustSaying.UnitTests/JustSayingBus/GivenAServiceBus.cs

```diff
@@ -50,6 +52,7 @@ public virtual Task DisposeAsync()
     protected virtual void Given()
     {
         Config = Substitute.For<IMessagingConfig>();
+        MessageReceivePauseSignal = Substitute.For<IMessageReceivePauseSignal>();
```

Comment:
I wonder if these fakes for tests need to be substitutes instead of using a plain `MessageReceivePauseSignal`? It's a fair bit more per-test setup if you wanted to do anything with it, instead of just using the regular implementation.

Reply:
It was more for consistency with `Config`, but I can change it to the regular implementation